go.mod

```diff
@@ -45,6 +45,7 @@ require (
 )
 
 replace (
+	github.com/apache/yunikorn-core v0.0.0-20230222073958-e80580b1a67f => github.com/elihschiff/yunikorn-core v0.0.0-20230323143210-256d1c68ca76
```

Comment:
I am leaving this here for now but I will update the go.mod/go.sum files once the other PR is merged. 

DO NOT MERGE this pr until after that point

Reply:
This PR now must be merged after https://github.com/apache/yunikorn-core/pull/523 is merged which solves the core issue limiting cluster autoscaling from working.

Reply:
please remove the go.mod and go.sum updates, the change can be committed without the update.

Reply:
👍 I removed the go.mod and go.sum changes